frontend/test/specs/event/eventControllerSpec.js

```diff
@@ -78,6 +82,8 @@
             };
         });
 
+        spyOn(Utils, 'isMobileScreen').and.returnValue(true);
```

Comment:
create an $onInit function in the controller instated of the main function there.
So you can test it properly.
And test when is mobile screen or not.